custom-templates/priv_instances_public_nlb_ssl/neo4j-nlb-priv-ssl.template.yaml

```diff
@@ -0,0 +1,857 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Neo4j v5 / NLB / Non Routable (Private) Instances
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+          default: "Neo4j Configuration"
+        Parameters:
+          - GraphDatabaseVersion
+          - InstallGraphDataScience
+          - GraphDataScienceLicenseKey
+          - InstallBloom
+          - BloomLicenseKey
+          - Password
+      - Label:
+          default: "Infrastructure Configuration"
+        Parameters:
+          - CertificateARN
+          - SSLDomain
+          - NumberOfServers
+          - InstanceType
+          - DiskSize
+          - BastionSSHCIDR
+          - SSHKeyName
+
+    ParameterLabels:
+      SSLDomain:
+        default: Domain for SSL/TLS
+      CertificateARN:
+        default: TLS/SSL Certificate ARN
+      GraphDatabaseVersion:
+        default: Graph Database Version
+      InstallGraphDataScience:
+        default: Install Graph Data Science
+      GraphDataScienceLicenseKey:
+        default: Graph Data Science License Key
+      InstallBloom:
+        default: Install Bloom
+      BloomLicenseKey:
+        default: Bloom License Key
+      Password:
+        default: Password
+      NumberOfServers:
+        default: Number of Servers
+      InstanceType:
+        default: Instance Type
+      DiskSize:
+        default: Disk Size
+      BastionSSHCIDR:
+        default: Bastion Host SSH CIDR  
+      SSHKeyName:
+        default: SSH Key Name 
+
+Parameters:
+  SSLDomain:
+    Description: The fully qualified domain name which will be used for SSL/TLS
+    Type: String
+
+  CertificateARN:
+    Description: The ARN of the certificate to be used with the NLB Listeners
+    Type: String
+
+  GraphDatabaseVersion:
+    Description: Neo4j Graph Database Version
+    Type: String
+    Default: '5.1.0'
+
+  InstallGraphDataScience:
+    Description: Install Graph Data Science
+    Type: String
+    Default: 'False'
+    AllowedValues:
+      - 'True'
+      - 'False'
+
+  GraphDataScienceLicenseKey:
+    Description: License Key for Graph Data Science (License keys will be sent to and stored by Neo4j. This information will only be used for the purposes of product activation.)
+    Type: String
+    Default: 'None'
+
+  InstallBloom:
+    Description: Install Bloom
+    Type: String
+    Default: 'False'
+    AllowedValues:
+      - 'True'
+      - 'False'
+
+  BloomLicenseKey:
+    Description: License Key for Bloom (License keys will be sent to and stored by Neo4j. This information will only be used for the purposes of product activation.)
+    Type: String
+    Default: 'None'
+
+  Password:
+    Description: Password for Neo4j
+    Type: String
+    MinLength: 6
+    NoEcho: true
+
+  NumberOfServers:
+    Description: Must be 3 or greater to form a cluster
+    Type: Number
+    Default: 3
+    AllowedValues:
+      - 1
```

Comment:
Should remove this value as it conflicts with the description. Is `AllowedValues` presented as a dropdown in Cloudformation, then this value of `1` may stand out to the user